### PR TITLE
Display board

### PR DESCRIPTION
**Pull request by scottyplunkett**

### CODE REVIEW

File: tictactoe/test/format_board_tests.erl [hunk 1]
```diff
@@ -0,0 +1,17 @@
+-module(format_board_tests).
+-include_lib("eunit/include/eunit.hrl").
+-import(format_board, []).
+
+get_board_length_test() ->
+  [{ "gets the length of a board",
+     ?assertEqual(9,format_board:get_board_length([1,2,3,4,5,6,7,8,9])),
+     ?assertNotEqual(8,format_board:get_board_length([1,2,3,4,5,6,7,8,9]))}].
```
Comment (NicoleCarpenter) [inline tictactoe/test/format_board_tests.erl]: This is redundant

File: tictactoe/test/format_board_tests.erl [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+get_board_row_length_test() ->
+	[{ "gets the length of a board row",
+		?assertEqual(3,format_board:get_board_row_length(9)),
+		?assertNotEqual(4,format_board:get_board_row_length(9))}].
```
Comment (NicoleCarpenter) [inline tictactoe/test/format_board_tests.erl]: You don't need this if you have already established equality on the line above.

File: tictactoe/src/format_board.erl [hunk 3]
```diff
@@ -0,0 +1,33 @@
+-module(format_board).
+-export([	get_board_length/1,
+					get_board_row_length/1,
+					split_board_into_rows/1,split_board_into_rows/2]).
+
+get_board_length(Board) -> length(Board).
+
+get_board_row_length(BoardLength) -> trunc(math:sqrt(BoardLength)).
+
+split_board_into_rows(Board) ->
```
Comment (NicoleCarpenter) [inline tictactoe/src/format_board.erl:10]: Interesting. It works 👍

**Review by NicoleCarpenter — CHANGES_REQUESTED** [review]

File: tictactoe/src/console_io.erl [hunk 4]
```diff
@@ -1,11 +1,18 @@
...
+					greet_user/0,
+					display_board/1]).
 -import(instructions, [greet/0]).
 
 
 display(Message) ->
   io:fwrite(Message).
 
+display_board([]) -> board;
+display_board([RowToPrint | RemainingRows]) ->
```
Comment (paulwpagel) [inline tictactoe/src/console_io.erl]: Where is the test for this function?

**Comment by scottyplunkett** [comment]
Changes implemented... I believe the reason for my original redundancy was an attempt at testing the test to make sure it wasn't giving false positives.

File: tictactoe/src/console_io.erl (hunk 4 above)
Comment (scottyplunkett) [reply tictactoe/src/console_io.erl]: This one I wasn't sure how to test, how would you recommend I test something like this?

**Review by NicoleCarpenter — APPROVED** [review]

**Pull request merged**